jelares/portfolio/target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Mon Jun 22 16:57:27 UTC 2020
+#Tue Jun 23 16:01:07 UTC 2020
```

Comment:
Maybe this file could be added to the repo's .gitignore?

Reply:
target/ is already in the gitignore, I am not sure why it still tracks the files. I'll be more careful in the future not to include them